modules/dashboard/widgets/dashboard-notepad.php

```diff
@@ -81,14 +78,14 @@ public function notepad_widget() {
 		$current_note = ( ! empty( $posts[0]->post_content ) ) ? $posts[0]->post_content : '';
 		$current_id = ( ! empty( $posts[0]->ID ) ) ? $posts[0]->ID : 0;
 		$current_post = ( ! empty( $posts[0] ) ) ? $posts[0] : false;
-
+		
```

Comment:
The whitespace should be part of a separate commit as per https://vip.wordpress.com/documentation/creating-good-changesets/